### ISSUE TITLE
Outlook Account Not Connecting

### ISSUE THREAD

**Issue by Loshay1** [Oxygem/Kanmail#216]
[**Describe](url) the bug**

### I been trying to log in using the Outlook option, however, I keep getting an error show in the screenshot.

![CleanShot 2023-03-23 at 01 47 29](https://user-images.githubusercontent.com/122305641/227125304-4fab7f82-3533-4f70-88ae-2c8ef7000a52.png)

I then tried adding the account manually by using the settings below, and also tried changing the options, however, I still have not been able to log in. My machine's specs are in the screen shot below. Can someone please advise on how I can get my account logged in?

- Office 365 IMAP settings
- Server: [outlook.office365.com](https://outlook.office365.com/).
- Port: 995.
- Encryption: SSL/TLS.
- Authentication: Required.

- Office 365 SMTP settings
- Server name: [smtp.office365.com](https://smtp.office365.com/).
- Port: 587.
- Encryption method: SSL/TLS.
- SMTP authentication: ticked.
- Authentication: your password.

**Screenshots**
![CleanShot 2023-03-23 at 01 52 00](https://user-images.githubusercontent.com/122305641/227125852-9819e29d-c320-4e4d-b611-bbceb09c273d.png)

![CleanShot 2023-03-22 at 11 55 04](https://user-images.githubusercontent.com/122305641/227124639-9cb91f5d-52d3-43c9-a2e4-fcdf9ca18499.png)

![CleanShot 2023-03-22 at 11 55 18](https://user-images.githubusercontent.com/122305641/227124651-9ce7232f-a989-4862-930b-ea21877f3eee.png)

**Comment by Loshay1** [comment]
Can anyone help on this?

**Comment by chranmat** [comment]
@Loshay1 Office365 does not, by default, enable IMAP/SMTP. Have you enabled this in your Office365 tenant?

@Fizzadar This app looks great, but as most professional users have Office365 or G-Suite, directly integrating these two using modern protocols (not IMAP/SMTP) would boost the usability. Is there any plan for this?

**Comment by Loshay1** [comment]
@chranmat yes, I have this enabled but still can’t seem to login. Have you been able to get it to work?

Thank You,

Larab Mohsin, Esq.




This electronic mail transmission may contain confidential or privileged information.  The information is intended to be used solely by the recipient(s) named above. If you are not an intended recipient or if you believe you have received this transmission in error, please notify the sender by replying, and delete the message without copying or disclosing it.

**Comment by chranmat** [comment]
@Loshay1 No, I have yet to try it because it does not support direct integration and modern authentication with Microsoft. Even if I asked you if you had enabled this, I would not recommend you do so due to security concerns.

Modern authentication is a crucial feature for organizations using Microsoft 365, providing a more secure way to access cloud-based services. With modern authentication, users can authenticate via an access token rather than relying solely on username and password credentials. This allows for better security controls, as access tokens can be revoked or expired as needed. Additionally, modern authentication enables the use of multi-factor authentication (MFA), which adds an extra layer of protection by requiring users to verify their identity through an additional factor, such as a fingerprint or code sent to their phone. MFA significantly reduces the risk of account compromise, as even if a password is compromised, an attacker still needs access to the user's second factor. Therefore, using modern authentication with MFA is an essential step towards securing sensitive data and ensuring the safety of Microsoft 365 users.

**Comment by mcloesmeijer** [comment]
Same challenge : office365 and cannot login. And no possibility to change admin settings as I am using corporate accounts and IT won't let me near the admin panel.

**Comment by ghost** [comment]
I believe this related. https://github.com/Oxygem/Kanmail/issues/179 It would be great if modern authentication with Microsoft could be added. I'd love to use this program in my regular workflow but can't add 365 accounts to it because of authentication security.

**Comment by Fizzadar** [comment]
v2 now supports Outlook via OAuth!
